cypress/components/preview/preview.cy.tsx

```diff
@@ -9,13 +10,9 @@ import { assertCssValueIsApproximately } from "../../support/component-helper/co
 const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
 const pixelsData = [256, 275, 300];
 
-const PreviewComponent = ({ ...props }) => {
-  return <Preview loading {...props} />;
-};
-
 context("Tests for Preview component", () => {
   describe("should check Preview component properties", () => {
-    it.each(pixelsData)(
+    it.each([...pixelsData])(
```

Comment:
Question: Do we need to spread these as they're already arrays? Wondering if the below would work for all these `it.each` blocks?

```suggestion
    it.each(pixelsData)(
```

Reply:
if it is not assigned `as const` it should not be `[...pixelData]`